tests/NotPresentValidatorTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Saritasa\Laravel\Validation\Tests;
+
+use Illuminate\Config\Repository;
+use Illuminate\Validation\Validator;
+use Orchestra\Testbench\TestCase as BaseTestCase;
+use Saritasa\Laravel\Validation\FluentValidationServiceProvider;
+use Saritasa\Laravel\Validation\FluentValidatorFactory;
+use Saritasa\Laravel\Validation\Rule;
+use Saritasa\Laravel\Validation\Rules\NotPresent;
+
+class NotPresentValidatorTest extends BaseTestCase
+{
+    /**
+     * Validator factory
+     *
+     * @var FluentValidatorFactory
+     */
+    protected $validator;
+
+    protected function getPackageProviders($application)
+    {
+        return [
+            FluentValidationServiceProvider::class
+        ];
+    }
+
+    protected function getPackageAliases($app)
+    {
+        return [
+            'config' => Repository::class,
+        ];
+    }
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->validator = $this->app['validator'];
+    }
+
+    public function testNotPresentValid()
+    {
+        $validator = $this->validator->make([
+            'field' => null,
+        ], [
+            'field' => 'not_present',
+        ]);
+        $result = $validator->passes();
+        $this->assertTrue($result);
+    }
+
+    public function testNotPresentValid2()
+    {
+        $result = $validator = $this->validator->make([], [
+            'field' => 'not_present',
+        ])->passes();
+        $this->assertTrue($result);
+    }
+
+    public function testNotPresentInvalid()
+    {
+        $result = $validator = $this->validator->make([
+            'field' => 'something',
+        ], [
+            'field' => 'not_present',
+        ])->passes();
+        $this->assertFalse($result);
+    }
+
+    public function testNotPresentValidWithFluentWay()
+    {
+        $result = $this->validator->make([
+            'field' => null,
```

Comment:
Field 'field' is presented in arry of fields but test passed. This is unexpected behaviour and differs from rule description. Now rule works not as 'not_present' but like 'should_be_empty'

Reply:
I have just committed new code. It fixed this case.